runtime/include/tt/runtime/detail/ttnn.h

```diff
@@ -59,6 +60,8 @@
 
 namespace tt::runtime::ttnn {
 
+constexpr std::size_t kL1SmallSize = 1 << 15;
```

Comment:
Can you add a comment explaining what this is for?

Reply:
This is a temporary default L1Smallsize we will be using until we can allocate L1 per op more specifically. The original default is 0, which causes some buffers to not be created at all. I was running into issues with that while bringing up conv2d. @nsmithtt  anything to add?

Reply:
It's a special allocator flag in TTNN that *must* be programmed during device creation when using convs.  Conv op specifically uses this region to store sideband indexing info for the op, it is kind of a special case in this regard.